### PR TITLE
Feat/manage prs

### PR DESCRIPTION
**Pull request by jjacque**
This PR contain actions that allow to:
  - list pull requests with filtering on base or head branch, basic regex on title
  - review (for now approval only) a PR given its github id
  - merge a PR given its github id

### CODE REVIEW

File: actions/review_pull.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: review_pull
+runner_type: python-script
+description: Retrieve information about a particular Github issue.
```
Comment (LindsayHill) [inline actions/review_pull.yaml]: Looks like metadata update needed here

File: actions/review_pull.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    type: "string"
+    description: "Repository name."
+    required: true
+  pull_id:
+    type: "integer"
+    description: Pull Request id
+    required: true
+  message:
+    type: "string"
+    description: Pull Request id
```
Comment (LindsayHill) [inline actions/review_pull.yaml]: Description update needed here too

**Review by LindsayHill — APPROVED** [review]
Thanks for the PR! Generally looks good, couple of minor description field updates needed. If you could update those, add an entry to CHANGES.md, and bump the version in `pack.yaml`, then I'll merge it

File: actions/review_pull.yaml (hunk 1 above)
Comment (jjacque) [reply actions/review_pull.yaml]: Hi, thanks for your answer, i made the changes for all descriptions (action and parameter)

**Comment by jjacque** [comment]
I also made a small change to the review_pull action, to also allow other events than APPROVE and bumped the version.

**Comment by LindsayHill** [comment]
Excellent, thanks for that

**Pull request merged**